database/migrations/2016_04_13_224615_create_project_comments_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateProjectCommentsTable extends Migration
```

Comment:
:+1: for migrations. Would be nice also if you could seed the table with some default entries (or the command like I did). Not necessary though